api/balancehelper.py

```diff
@@ -138,7 +138,7 @@ def get_bulkbalancedata(addresses):
                              atx.propertyid) s2
                         on s1.propertyid=s2.propertyid) f1
                      inner join smartproperties sp
-                     on f1.propertyid=sp.propertyid and (sp.protocol='Mastercoin' or sp.protocol='Bitcoin')
+                     on f1.propertyid=sp.propertyid and (sp.protocol='Omni' or sp.protocol='Bitcoin')
```

Comment:
@Nevtep same here, lets not remove the old mastercoin from this query, instead append Omni as another 'or' statement. this will preserve some backwards compatibility until the databases are fully converted/updated

sp.protocol='Omni' or sp.protocol='Mastercoin' or sp.protocol='Bitcoin'